ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/port-channel-interfaces.yml

```diff
@@ -55,23 +55,6 @@ port_channel_interfaces:
       - LEAF_PEER_L3
       - MLAG
 
-  - name: Port-Channel10
```

Comment:
There are many changes in the molecule data which makes it harder to review. I would prefer to keep data 1:1 so we see no change in the generated configs/docs.

Reply:
Reverted the molecule changes. hopefully it's ok now.